README.md

```diff
@@ -390,6 +390,8 @@ var gce = gcloud.compute({
 var zone = gce.zone('us-central1-a');
 var name = 'ubuntu-http';
 
+// Optionaly you can specify `{ os: 'your-project-id-or-name/ubuntu'}`
```

Comment:
We can remove this here. The readme is mostly a place for our simplest use cases.